include/log.h

```diff
@@ -1,22 +1,54 @@
 #ifndef _SWAY_LOG_H
 #define _SWAY_LOG_H
+
 #include <stdbool.h>
-#include <wlr/util/log.h>
+#include <stdarg.h>
+#include <string.h>
+#include <errno.h>
+
+typedef enum {
+	SWAY_SILENT = 0,
+	SWAY_ERROR = 1,
+	SWAY_INFO = 2,
+	SWAY_DEBUG = 3,
+	SWAY_LAST,
```

Comment:
These could be `L_*` instead. `SWAY_LAST` is not a very good name.

Reply:
I'm now using SWAY_LOG_IMPORTANCE_LAST, similar to WLR_LOG_IMPORTANCE_LAST.